src/converter.js

```diff
@@ -32,6 +44,11 @@ function genValuePartial_fromObject(gen, field, fieldIndex, prop) {
                     ("break");
             } gen
             ("}");
+        } else if (WRAPPER_TYPES.indexOf(field.type) !== -1) {
+            gen
+            ("if(typeof d%s===\"object\" && d%s!==null && d%s.value!==undefined)", prop, prop, prop)
+                ("throw TypeError(%j)", field.fullName + ": wrapped value not expected")
+            ("m%s=types[%i].fromObject({value: d%s})", prop, fieldIndex, prop)
```

Comment:
Example difference in our output:

```
-                        if (typeof d.creationTimestamp !== "object")
-                            throw TypeError(".mix.data.protobuf.EventHeader.creationTimestamp: object expected");
-                        m.creationTimestamp = $root.google.protobuf.UInt64Value.fromObject(d.creationTimestamp);
+                        if (typeof d.creationTimestamp == "object" && d.creationTimestamp !== null && d.creationTimestamp.value !== undefined)
+                            throw TypeError(".mix.data.protobuf.EventHeader.creationTimestamp: wrapped value not expected");
+                        m.creationTimestamp = $root.google.protobuf.UInt64Value.fromObject({ value: d.creationTimestamp });
```

(See lines 53-55 below to see what generated the previous version)